src/Raven.Studio/typescript/viewmodels/shell/about.ts

```diff
@@ -8,11 +8,16 @@ import accessManager = require("common/shell/accessManager");
 import forceLicenseUpdateCommand = require("commands/licensing/forceLicenseUpdateCommand");
 import getLatestVersionInfoCommand = require("commands/version/getLatestVersionInfoCommand");
 import getLicenseConfigurationSettingsCommand = require("commands/licensing/getLicenseConfigurationSettingsCommand");
+import clusterTopologyManager = require("common/shell/clusterTopologyManager");
+import appUrl = require("common/appUrl");
 
 class about extends viewModelBase {
 
     accessManager = accessManager.default.aboutView;
     
+    clusterViewUrl = appUrl.forCluster();
+    passiveNode = ko.pureComputed(() => clusterTopologyManager.default.nodesCount() === 0);
```

Comment:
isn't better to use: https://github.com/ravendb/ravendb/blob/135743f79f7a52bfb0f747c432689039f8ed4f15/src/Raven.Studio/typescript/models/database/cluster/clusterTopology.ts#L10 
from 
https://github.com/ravendb/ravendb/blob/v5.0/src/Raven.Studio/typescript/common/shell/clusterTopologyManager.ts#L13

Reply:
done